.eslintrc.json

```diff
@@ -42,6 +42,12 @@
           "ClassDeclaration": true
         }
       }
-    ]
+    ],
+    "prefer-destructuring":"off",
+    "no-console":"off",
```

Comment:
```suggestion
```
This rule ensure that no console log are shipped into production. Mostly console is used for debbugging purpose, if want to log some information you can use loggers like winston
https://github.com/winstonjs/winston